### ISSUE TITLE
Updating Forge Mapping

### ISSUE THREAD

**Issue by EoD** [Mithion/ArsMagica2#1297]
The newest version of ForgeGradle allows to specify an updated version of the obfuscation mappings ``stable_12``.
https://github.com/Mithion/ArsMagica2/blob/master/build.gradle#L27:
```gradle
minecraft {
    version = "1.7.10-10.13.4.1448-1.7.10"
    runDir = "run"
    mappings = "stable_12"
}
```

This adds a lot of unknown methods/variables as well as updates the naming scheme to make a potential update to 1.8 easier.
If there's enough interest, I would start this process in a separate branch. This will be a tedious task with a lot of rebasing, but it's worth it imho.

**Comment by Mithion** [comment]
I'm less concerned about a 1.8 update - personally I would like to focus on getting this next milestone complete first so that we can get a release out for 1.7.10.

**Comment by EoD** [comment]
My main concern was less obfuscated fields/methods and more readable ones. The 1.8 compatibility is just a side-effect.

**Comment by Kolatra** [comment]
When I have used these mappings in my own mods, I've noticed compatibility issues with other mods in the dev environment that don't use the latest mappings, causing a bunch of NoSuchMethodErrors.

**Comment by Mithion** [comment]
From what I can see of the commit, many of the changes are simply method parameters, which I am not really concerned about.

I'd rather aim for compatibility, as thus far we've been able to infer what the methods are by context.

**Comment by EoD** [comment]
For me the dev build of AM2 never worked, so I assumed there will be no non-obfuscated version of AM2.

**Comment by Mithion** [comment]
I never released a dev build before.

Generally what I did was to just pop CodeChickenCore into the mods folder in the dev environment.  It allows obfuscated jars to be used in dev.  Since many other mods do not release a deobf version, I found this to be more helpful to point people to, as it will help them use many other mods in dev.

**Comment by EoD** [comment]
Even though this will drive us off-topic here: As far as I can say the ``specialsource.jar`` can deobfuscate those jars for you if you really need it :)

**Comment by EoD** [comment]
This issue can be closed as #1303 has been rejected.
